fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/jbatch/bulkdata/export/patient/ChunkReader.java

```diff
@@ -175,10 +175,15 @@ protected void fillChunkDataBuffer(List<String> patientIds) throws Exception {
                         searchContext.setPageSize(pageSize);
                         searchContext.setPageNumber(compartmentPageNum);
                         FHIRTransactionHelper txn = new FHIRTransactionHelper(fhirPersistence.getTransaction());
-                        txn.enroll();
                         FHIRPersistenceContext persistenceContext = FHIRPersistenceContextFactory.createPersistenceContext(null, searchContext);
-                        List<Resource> resources = fhirPersistence.search(persistenceContext, resourceType).getResource();
-                        txn.unenroll();
+
+                        List<Resource> resources;
+                        txn.begin();
+                        try {
+                            resources = fhirPersistence.search(persistenceContext, resourceType).getResource();
+                        } finally {
+                            txn.end();
+                        }
```

Comment:
not sure if this can work well, need to be verified by the minio enabled build pipeline.